packages/xo-web/src/xo-app/vm/tab-disks.js

```diff
@@ -248,6 +248,14 @@ const parseBootOrder = bootOrder => {
   return order
 }
 
+const WarningSr = ({ n }) => (
+  <div>
+    <span className='text-danger'>
+      <Icon icon='alarm' /> {_('warningSr', { n })}
+    </span>
+  </div>
+)
```

Comment:
If having a component for this becomes useless at some point, you may want to inline it.

Reply:
I used a constant.